qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1996,33 +2050,35 @@ def get_queue(
         TODO: check what query keys are needs
         Parameters
         ----------
-        id : list
+        id : Optional[List[str]], optional
             Ids of the tasks
-        Hash_index
-        status : bool, default is None (find all)
+        Hash_index: Optional[List[str]], optional,
+            hash_index of service, not used
+        program, list of str or str, optional
+        status : Optional[bool], optional (find all)
             The status of the task: 'COMPLETE', 'RUNNING', 'WAITING', or 'ERROR'
-        base_result: str (optional)
+        base_result: Optional[str], optional
             base_result id
-        include : list/set/tuple of keys, default is None
+        include : Optional[List[str]], optional
```

Comment:
Need need for `optional` on `Optional[...]` objects.

Again, I really recommend [AutoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) or similar to do this for you correctly. Automating this will save you a ton of time and prevent small issues.

Reply:
I have installed the extension, but did not understand the problem here, is the Optional typing inside incorrect?

Reply:
Hmm, it looks like sometimes the module has it implied while other times it adds it explicitly. I guess this is fine, seems to render ok oddly enough.